docs/link-manager.md

```diff
@@ -0,0 +1,43 @@
+## Overview
+
+Use the Link Manager instead of an internal router to: 
+
+- route inside micro frontends 
+- reflect the route and 
+- keep the navigation state in Luigi
+
+Here is a code sample and a definition of the different parameters:
+
+````
+// navigation
+LuigiClient.linkManager().navigate('/overview')
+LuigiClient.linkManager().navigate('users/groups/stakeholders')
+LuigiClient.linkManager().fromClosestContext().navigate('/users/groups/stakeholders')
+LuigiClient.linkManager().fromContext('project').navigate('/settings')
+ 
+ 
+// additionally send parameters to route
+LuigiClient.linkManager().fromClosestContext().withParams({foo: 'bar'}).navigate('settings')
+ 
+ 
+// feature: preserve view to go back afterwards
+LuigiClient.linkManager().fromContext('project').navigate('/settings', null, true) // preserve view
+LuigiClient.linkManager().hasBack()
+LuigiClient.linkManager().goBack({foo: 'bar})
+````
+
+#### Navigation
+
+- **navigate(path)** -  Contains either a full absolute path or a relative path without a leading slash that uses the currently active route as a base. This is classical navigation.
+- **fromClosestContext()** - Navigates from the closest parent node that has `navigationContext` declared in its navigation configuration. The node is usually one with a dynamic path segment.
```

Comment:
1. `Node that has the **navigationContext**` ...
2. `There is usually one Node with a dynamic path segment.` -> is that what you mean?

Reply:
1. node is written small, right?  `Navigates from the closest parent node that has the ****navigationContext** declared in its navigation configuration.`
2. It means that navigationContext is used only/always in combination with a dynamic path segment.

Reply:
No, Node should be capitalized throughout the whole documentation